app/views/finders/show.html.erb

```diff
@@ -16,86 +16,32 @@
 
 <% content_for :meta_title, finder.name %>
 
-<% if finder.show_phase_banner? %>
-  <%= render 'govuk_publishing_components/components/phase_banner', phase: finder.phase, message: finder.phase_message%>
-<% end %>
-
-<% if @breadcrumbs.breadcrumbs %>
-  <%= render 'govuk_publishing_components/components/breadcrumbs', breadcrumbs: @breadcrumbs.breadcrumbs %>
-<% else %>
-  <%= render 'govuk_publishing_components/components/contextual_breadcrumbs', content_item: @raw_content_item, prioritise_taxon_breadcrumbs: finder.eu_exit_finder?  %>
-<% end %>
-
-<% if finder.government? %>
-  <%= render 'govuk_publishing_components/components/government_navigation', active: finder.government_content_section %>
+<% if finder.topic_finder? && !finder.all_content_finder? %>
+  <% content_for :body_classes, "full-width" %>
+  <% inverse = true %>
 <% end %>
 
 <form method="get" action="<%= finder.slug %>" class="js-live-search-form">
   <input type="hidden" name="parent" value="<%= @parent %>">
 
-  <header class="govuk-grid-row">
-    <div class="govuk-grid-column-two-thirds">
-      <% if finder.all_content_finder? %>
-        <% label_text = capture do %>
-          <h1 class="app-c-search-page-heading">Search<span class="govuk-visually-hidden"> all content</span></h1>
-        <% end %>
-        <div id="keywords" class="app-patch--search-input-override">
-          <%= render "govuk_publishing_components/components/input", {
-            controls: "js-search-results-info",
-            id: "finder-keyword-search",
-            label: {
-              text: label_text,
-            },
-            name: "keywords",
-            search_icon: true,
-            type: 'search',
-            value: @results.user_supplied_keywords,
-          } %>
-        </div>
-      <% else %>
-        <%= render partial: 'govuk_publishing_components/components/title', locals: {
-            title: finder.name
-        } %>
-      <% end %>
-
-      <% if finder.page_metadata.any? %>
-        <%= render 'govuk_publishing_components/components/metadata', page_metadata(finder.page_metadata) %>
-      <% end %>
-
-      <% if finder.summary %>
-        <div class="metadata-summary">
-          <%= render 'govuk_publishing_components/components/govspeak', content: finder.summary.html_safe %>
-        </div>
-      <% end %>
-    </div>
-
-    <% if finder.logo_path %>
-      <div class="finder-logo govuk-grid-column-one-third">
-        <%= image_tag finder.logo_path, class: "finder-logo__image" %>
-      </div>
+  <% if finder.topic_finder? && !finder.all_content_finder? %>
+    <%= render "govuk_publishing_components/components/inverse_header", {
+      full_width: true
+    } do %>
+      <%= render partial: 'show_header', locals: { inverse: inverse } %>
     <% end %>
+  <% else %>
+    <%= render partial: 'show_header', locals: { inverse: inverse } %>
```

Comment:
It seems like this line is duplicated above. Is there a way we can take this out of the if/else statement?

Reply:
Not easy to do, lets skip it!